python/tests/test_cb.json

```diff
@@ -0,0 +1,52 @@
+[
+    {
+        "data_func": "generate_cb_data",
+        "data_func_args": {
+            "num_examples": 100,
+            "num_features": 1,
+            "num_action": 1
+        },
+        "assert_functions": [
+            {
+                "assert_func": "assert_loss",
```

Comment:
better to expose both reward and expected loss values as parameters to this config file